test/awss3.jl

```diff
@@ -0,0 +1,47 @@
+function minio_server(body, dirs=[mktempdir()]; address="localhost:9005")
+    server = Minio.Server(dirs; address)
+
+    try
+        run(server; wait=false)
+        sleep(0.5)  # give the server just a bit of time, though it is amazingly fast to start
+
+        config = MinioConfig(
+            "http://$address"; username="minioadmin", password="minioadmin"
+        )
+        body(config)
+    finally
+        # Make sure we kill the server even if a test failed.
+        kill(server)
+    end
+end
+
+@testset "AWSS3 usage" begin
+    minio_server() do config
+        s3_create_bucket(config, "test-bucket")
+
+        file_format = "lpcm.zst"
+        file_path = S3Path("s3://test-bucket/prefix/samples.$(file_format)"; config)
+        recording_uuid = uuid4()
+        start = Second(0)
+
+        info = SamplesInfoV2(sensor_type="eeg",
+            channels=["a", "b"],
+            sample_unit="unit",
+            sample_resolution_in_unit=1.0,
+            sample_offset_in_unit=0.0,
+            sample_type=Int16,
+            sample_rate=100.0)
+        samples = Samples(zeros(sample_type(info), 2, 300), info, true)
```

Comment:
if you're gonna test for equality with the sub-span below, better make this random values instead of zeros

```suggestion
        samples = Samples(rand(sample_type(info), 2, 300), info, true)
```

Reply:
oops, copied from another test and didn't think it through!